introToFunctions/task2.js

```diff
@@ -0,0 +1,7 @@
+function isUserAdult(...ages) {
+    for (const age of ages) {
+        const isAdult=age>= 18
+        console.log(`Is this person ${age} years old an adult? ${isAdult}`);
+    }
+}
+isUserAdult(15, 25);
```

Comment:
your function should have only one parameter and return boolean value according to the task